Documentation/ceph-quickstart.md

```diff
@@ -96,6 +96,55 @@ rook-ceph-osd-1-5b96b56684-r7zsp       1/1       Running     0          1m
 rook-ceph-osd-prepare-mynode-ftt57     0/1       Completed   0          1m
 ```
 
+# Support for authenticated docker registries
+
+If you want to use an image from authenticated docker registry (e.g. for local development on ceph-agent), you'll need to 
```

Comment:
I think instead of `for local development ...` something like `image mirror/cache` is more applicable to the wider "quickstart audience"